server/rest/src/main/java/org/infinispan/rest/resources/CacheResourceV2.java

```diff
@@ -260,7 +259,7 @@ public Invocations getInvocations() {
    }
 
    private CompletableFuture<RestResponse> createCache(RestRequest request) {
-      NettyRestResponse.Builder responseBuilder = new NettyRestResponse.Builder().status(NO_CONTENT);
```

Comment:
Why the change? AFAICT this will make create cache return 200 instead of 204?

Reply:
No, it seems to be set always to `200` implicitly later with `responseBuilder.status(OK);` Please see L275, L285 and L307
it's just redundant call

Reply:
ok, you are right, but it should be 204 in this case, too late to fix it now